### ISSUE TITLE
pipelines: pas deux fois la meme réponse lorsque on a beaucoup d'overlap

### ISSUE THREAD

**Issue by guillim** [etalab-ia/piaf-ml#69]
piaf-ml: dans la pipelines, faire en sorte que il n'y ai pas deux fois la meme réponse lorsque on a beaucoup d'overlap

**Comment by guillim** [comment]
same issue on CRPA
![Screenshot 2021-06-17 at 15 42 22](https://user-images.githubusercontent.com/13087916/122408965-0ebf1000-cf83-11eb-9bec-118b386a744a.png)

**Comment by guillim** [comment]
it is getting a prio for our friends Cindy et Perica

**Comment by Rob192** [comment]
I just read on Haystack that their reader is removing duplicate spans (I am not sure weither or not we can consider our exemples here as duplicates though ...) 

> ## Deeper Dive: FARM vs Transformers
> 
> Apart from the model weights, Haystack Readers contain all the components found in end-to-end open domain QA systems. This includes tokenization, embedding computation, span prediction and candidate aggregation. While the handling of model weights is the same between the FARM and Transformers libraries, their QA pipelines differ in some ways. The major points are:
> 
> The TransformersReader will sometimes predict the same span twice while duplicates are removed in the FARMReader
> The FARMReader currently uses the tokenizers from the HuggingFace Transformers library while the TransformersReader uses the tokenizers from the HuggingFace Tokenizers library
> Start and end logits are normalized per passage and multiplied in the TransformersReader while they are summed and not normalised in the FARMReader
> If you’re interested in the finer details of these points, have a look at this GitHub comment.

**Comment by guillim** [comment]
So you think we shall use their FARM or you are suggesting to have a look at their implementation ?

**Comment by guillim** [comment]
Better to create our own custom component to tackle this

**Comment by guillim** [comment]
https://github.com/etalab-ia/piaf-ml/issues/112

**Comment by guillim** [comment]
all published in production now
